lib/crystalball/helpers/diff.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module Crystalball
+  module Helpers
```

Comment:
Lets not have this namespace. It's too abstract. This is not a common use helper. This is part of our "business logic". 
Something like `Crystalball::FormattingChecker` sounds better.